### ISSUE TITLE
Collection run upgrade

### ISSUE THREAD

**Issue by berlingoqc** [usebruno/bruno#2971]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

The collection run feature is pretty simple.

I'll like to add the ability to `Load Data from File for Collection Run`. I see that this feature is in the golden edition.

If I create a PR with this feature in the opensource edition will it possibly get merged?

### Mockups or Images of the feature

Load CSV or similar kind of file for specific environment variables for the collection run. Also maybe would like to specify the list of request/folder to run in the collection run.

**Comment by Its-treason** [comment]
> If I create a PR with this feature in the opensource edition will it possibly get merged?

While I can't speak for Anoop directly, I'm pretty sure he will not accept a PR for this:

- This is a feature to incentive people to buy the golden edition
- The Golden Edition source code is proprietary and has its own implementation for this. The Open-source implementation would need to be merged into the golden edition

But this project is MIT licensed, so feel free to fork the project and implement this or any other feature on your own. I kinda did the same with my fork: [its-treason/bruno](https://github.com/its-treason/bruno)

**Comment by berlingoqc** [comment]
Yes I see, I would like to help with this project but I don't pay for software ever. I can donate money. I would like if the community could add features without consideration of the proprietary code. 

I guess it's the caveat of having an offline open-source tool that you want to monetize. You can't make the user pay for a hosting service so you have to make them pay for basic features required for more enterprise need
